docs/api/web-contents.md

```diff
@@ -1525,7 +1525,7 @@ End subscribing for frame presentation events.
 #### `contents.startDrag(item)`
 
 * `item` Object
-  * `file` String or `files` Array - The path(s) to the file(s) being dragged.
+  * `file` String[] | String - The path(s) to the file(s) being dragged.
```

Comment:
Parentheses not needed for `String[] | String`? (As opposed to `(String | null)` above.)

Reply:
Not required, the new parser doesn't need them 👍